src/Custom-sample.h

```diff
@@ -207,14 +207,22 @@
 
 //#define WEBPAGE_TEMPLATE_HIDE_HELP_BUTTON
 
+/*
+ #######################################################################################################
+   External CSS and JS
+ #######################################################################################################
+ */
+
+#define WEBSERVER_USE_CDN_JS_CSS  //When set, the followig URL will be used as a prefix for CSS, JS and favicon files, else they will be directly included into the HTML
+#define CDN_JS_CSS_URL           "https://cdn.jsdelivr.net/gh/letscontrolit/ESPEasy@mega-20210503/static/"
```

Comment:
This file is not used by all builds, so I don't think these `#define`'s should be here.

Reply:
I agree, but while this URL is used by SOME builds, and as this URL is subject to change for any new release (according to @TD-er previous commentss), I guess that it should be better as a global #define (easily change-able), rather than an URL coded inside a .cpp function.

This also allow me/others devs, to quickly force using an external URL on a local test server... as well as anyone wanting to have its own LOCALLY served medias for 1M devices (instead on relying on internet-only ressources), by overidding it in custom.h...

_Anyway : I'm of course at your disposal to change it according to your (all team) decisions...._